scripts/run_mypy_checks.py

```diff
@@ -61,8 +61,6 @@
     'core/domain/email_manager_test.py',
     'core/domain/email_services.py',
     'core/domain/email_services_test.py',
-    'core/domain/email_subscription_services.py',
-    'core/domain/email_subscription_services_test.py',
```

Comment:
Remove remaining files from the list.

Reply:
I accidentally removed this!
Now, this is fixed.